docs/getting-started/continuous_integration.md

```diff
@@ -136,6 +135,50 @@ jobs:
 {% endraw %}
 ```
 
+#### Branches and paths
+
+You can make CIFuzz to trigger only on certain branches or paths by following the
+instructions [here](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions).
+For example, the following code can used to trigger CIFuzz only on C/C++ code
+residing on master and release branches:
+
+```yaml
+name: CIFuzz
+on:
+  pull_request:
+    branches: 
+      - master
```

Comment:
Have you tested this? I wonder what it means to only test on master branch. Does it mean that only PRs merged into master are tested or that only commits to master are tested? The latter isn't useful since CIFuzz is only meant to run on PRs (which might be a bad assumption, but that's another story).

Reply:
> I wonder what it means to only test on master branch

As far as I know it means that it should be run when PRs are opened against the master branch of the base repository.

> CIFuzz is only meant to run on PRs (which might be a bad assumption, but that's another story).

CIFuzz can handle push events just fine. At least [`systemd`](https://github.com/systemd/systemd/blob/master/.github/workflows/cifuzz.yml) and [`curl`](https://github.com/curl/curl/blob/master/.github/workflows/fuzz.yml) run it every time the master branch gets updated.

Reply:
PRs opened against the set of branches that are specified. I will add the systemd example in this document, thanks a lot @evverx.